tests/e2e/tests/web/critical-path/rdi/dry-run-job.e2e.ts

```diff
@@ -0,0 +1,68 @@
+import { t } from 'testcafe';
+import { RdiInstance } from '../../../../pageObjects/components/myRedisDatabase/add-rdi-instance';
+import { updateColumnValueInDBTable } from '../../../../helpers/database-scripts';
+import { RdiInstancePage } from '../../../../pageObjects/rdi-instance-page';
+import { RdiInstancesListPage } from '../../../../pageObjects/rdi-instances-list-page';
+
+const rdiInstancePage = new RdiInstancePage();
+const rdiInstancesListPage = new RdiInstancesListPage();
+export const commonUrl = process.env.COMMON_URL || 'http://localhost:8080/integrate';
+const rdiTable = 'rdi';
+const resultMock = `{··"name":·"John",··"years":·123}`;
+const outputMock = 'Shirizli';
+
+const rdiInstance: RdiInstance = {
+    alias: 'testInstance',
+    url: 'url',
+    username: 'username',
+    password: 'password',
+    version: '1.2'
+};
+//skip the tests until rdi integration is added
+
+fixture.skip `Rdi dry run job`
+    .meta({ type: 'critical_path' })
+    // it will be removed
+    .page(commonUrl)
+    .beforeEach(async() => {
+        await t.maximizeWindow();
+
+    })
+    .afterEach(async() => {
+        // delete instances via UI
+        await rdiInstancesListPage.deleteAllInstance();
+    });
+test('Verify that user can use Dry run panel', async() => {
+    const instanceId = 'testId';
+    const instanceUrl = 'http://localhost:4000';
+    // const job = 'testJob';
+    const job = 'job2';
+
+    await rdiInstancesListPage.addRdi(rdiInstance);
```

Comment:
you can do it via API

Reply:
as I remember, I've already implemented it

Reply:
yes, thanks, switched to api methods